src/org/jitsi/impl/neomedia/rtp/RTPEncodingDesc.java

```diff
@@ -452,8 +452,10 @@ void update(RawPacket pkt, long nowMs)
         long ts = pkt.getTimestamp();
         FrameDesc frame = frames.get(ts);
 
+        boolean isPacketOfNewFrame;
```

Comment:
sorry i'm being nit-picky, but i think the 'first' part (`isFirstPacketOfNewFrame`) of the variable is important, right?